spec/cdn_source_spec.rb

```diff
@@ -29,6 +29,11 @@ def cleanup
         @path.join('.url').delete if @path.join('.url').exist?
       end
 
+      def print_dir(tag)
```

Comment:
this method appears tobe unused

Reply:
Yeah, but it's a useful one and it's in a spec. Is there a policy against helper code in tests?

Reply:
no, helper code is fine, but we shouldn't be commiting code that . never gets . run

Reply:
Well, where should one keep it for next time it's needed? Separate file?